javasphinx/htmlrst.py

```diff
@@ -233,7 +235,7 @@ def _process_text(self, node):
         return ''.join(node.strings)
 
     def _process(self, node):
-        if isinstance(node, basestring):
+        if isinstance(node, str):
```

Comment:
I think this change may cause problems with unicode strings in 2.x. Using the support from [python-future](http://python-future.org) for str/unicode handling throughout might be a good idea.